doc/tech/usecases.md

```diff
@@ -0,0 +1,194 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# CNTT Use Cases
+
+## Table of Contents
+ * [5.1 Edge](#5.1)
+   * [5.1.1 Executive Summary](#5.1.1)
+   * [5.1.2 Objective](#5.1.2)
+   * [5.1.3 Approach & Scope](#5.1.3)
+   * [5.1.4 Principle](#5.1.4)
+   * [5.1.5 Use cases](#5.1.5)
+   * [5.1.6 Terminologies](#5.1.6)
+   * [5.1.7 SDO interlocking](#5.1.7)
+   
+
+<a name="5.1"></a>
+## 5.1 Edge
+
+<a name="5.1.1"></a>
+### 5.1.1 Executive Summary
+
+Edge computing, a disruptive technology, that should be considered for adoption by Digital Service Providers (DSP) as it is tightly linked with a number of use cases that monetize 5G. Thus, operators, cloud service providers, content providers and application developers are increasingly focussed on Edge computing.
+
+Edge is considered as one of the main enablers to 5G use cases adoption. There are a number of 5G challenges that Edge will support solve so as to implement the 5G requirements defined by "ITU-2020 for 5G." Edge is required to support enhanced mobile broadband (eMBB), massive machine-type communications (mMTCs) and ultra-reliable low-latency communications (uRLLCs).
+
+With respect to monetization, Edge will help DSPs with their Return on investment (ROI) on 5G and cloud as a number of new services require the Edge. For example, services such as Cloud Gaming, Assisted Reality/Virtual Reality (AR/VR), 5G private networks, and vRAN.
+
+<a name="5.1.2"></a>
+### 5.1.2 Objective
+
+Based on CNTT’s goal and purpose is to develop a robust cloud infrastructure model and define a limited set of discrete architectures built on that model that can be tested and validated for use across the entire member community.
+
+Extend CNTT's Scope beyond the Regional and National Data Center cloud infrastructures to the Edge. The Edge is a disruptive use case where CNTT can add value especially as there are a number of scattered initiatives under various Standard Development Organisations (SDO) and open source communities."
+
+Edge Stream related activities :
```

Comment:
II think the lines that follow are supposed to be a list of bulleted entries, one per each line. If yes, then the bullet md "- " should be added to each line. If not, then they need to be added to form complete sentences with a period at the end of the sentence.

Example,

- To harmonize the work under Standard Development bodies or Open source communities
- To build Common Cloud infrastructure based on CNTT principles that can be consumed by any operator
- To build Cloud infrastructure that can scale over hundreds of thousands of nodes and cover the Edge Telco use cases that can help operators to monetize the NFV/SDN journey
- Modify the existing RM and RA-s so that they are aligned with the edge requirements

Reply:
Done , Thanks